examples/inference/lisa_smbhb/plot.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
+import pickle
+import numpy as np
+
```

Comment:
@WuShichao I think we don't need this file at all. Can't you replace this with a direct call to pycbc_inference_plot_posterior? Just put in the expected values directly.